tensorboard/webapp/testing/BUILD

```diff
@@ -9,12 +9,15 @@ licenses(["notice"])
 
 tf_ts_library(
     name = "initialize_testbed",
+    # Not an actual test but still needs to be named spec.ts since it is a
+    # top-level dep for spec_bundle().
     srcs = [
-        "initialize_testbed.ts",
+        "initialize_testbed.spec.ts",
```

Comment:
Seems strange that this is the only file whose name actually changed instead of the programmatic changing of all the other test files. Is it simply because it does not follow the "_test.ts" naming convention?

Reply:
Yes, exactly. I agree its a little weird.

Now that I think about it, I'd prefer to include a `s.endswith("_testbed.ts")` check in tf_ts_library. That solution is also a little imperfect but I think is more consistent with how we handle the other files.

Does that work for you?

Reply:
Yea I like this solution better.